### ISSUE TITLE
Open in Browser generates a folder with lots of files

### ISSUE THREAD

**Issue by alfredoq** [newsboat/newsboat#2389]
Good morning,

I noticed that opening a URL from newsboat with Google Chrome generate a '~google-chrome'  folder containing several subfolfders (i.e.  Default, WidevineCdm, GrShaderCache, etc). The content of this folder is updated every time a new feed is opened in the browser.

**My question is:** is it possible to prevent this data dumping to the folder, o at least purge it upon closing newsboat?

Thank you very much in advance for the support,

kind regards

Alfredo

**Comment by dennisschagt** [comment]
Hi Alfredo,
Newsboat does not create these kinds of folders, so I expect that Google Chrome creates the folder by itself.
Does Chrome also create the folder if you start it from outside of Newsboat?
Do you manually configure the browser in the newsboat config (via a `set browser …`  line) or is it configured via the `BROWSER` environment variable? (and do either of those contain extra command line arguments?)

**Comment by dennisschagt** [comment]
Alright, nice to hear you found the cause.

If you still want to store the database in a specific location, you could try using Newsboat's `--cache-file=/path/to/cache.db` option.
Be aware that you would then need to specify that every time (might be possible using an alias in your shell).

Just to be save, make sure you have backed up Newsboat's config/urls/cache files before starting to use that option.

Regardless, looks like the problem is resolved so I'm closing this issue.
Feel free to open a new issue (or reopen this issue) in case of related problems/questions.
